website/docs/quick-start/installation/modal/index.md

```diff
@@ -7,30 +7,39 @@
 First we import the components we need from `modal`.
 
 ```python
-from modal import Image, Stub, asgi_app, gpu
+import os
+from modal import Image, App, asgi_app, gpu, Volume
 ```
 
-Next, we set the base docker image version, which model to serve, taking care to specify the GPU configuration required to fit the model into VRAM.
+Next, we set the base Docker image version and specify which model to serve, using the `L4` GPU configuration to efficiently utilize VRAM.
```

Comment:
any reason a T4 GPU woundn't be sufficient?

Reply:
In response to your question about the sufficiency of a T4 GPU, during our testing phase, I encountered a significant number of HTTP 204 responses when using the T4 GPU. The exact cause of these issues is not clear, but it suggests that the T4 may not meet our resource or performance needs for tabby server.

Reply:
I add background on using Modal L4 GPU.

Reply:
Consider lowering the parallelism to something like 1/2? We still wanna have a low end GPU able to run a minimal setup of tabby.

Reply:
I'll try adjusting the parallelism to about 1/2 and see how it performs with a minimal setup of Tabby.